config/webpack/loaders/sass.js

```diff
@@ -6,6 +6,7 @@ module.exports = {
   use: ExtractTextPlugin.extract({
     fallback: 'style-loader',
     use: [
+      'thread-loader',
```

Comment:
https://webpack.js.org/guides/build-performance/#sass
> node-sass has a bug which blocks threads from the Node.js threadpool. When using it with the thread-loader set workerParallelJobs: 2.

Will this not affect to our case?